src/main/scala/pro/civitaspo/digdag/plugin/ecs_task/command/S3TmpStorage.scala

```diff
@@ -48,7 +48,7 @@ case class S3TmpStorage(location: AmazonS3URI, aws: Aws, workspace: Workspace, l
   }
 
   override def buildTaskCommand(mainScript: String): Seq[String] = {
-    Seq("sh", "-c", s"aws s3 cp ${location.toString}/$mainScript ./ && sh $mainScript")
+    Seq("sh", "-c", s"aws s3 cp ${location.toString}/$mainScript ./ && /bin/bash $mainScript")
```

Comment:
Please change `/bin/bash` to `bash` because `bash` is not necessarily located in `/bin/bash`.